Makefile

```diff
@@ -1,7 +1,7 @@
 # Package related
 TOOLSDIR=$(CURDIR)/hack/tools/bin
 BUILDDIR=$(CURDIR)/build/_output
-
+GO = go
```

Comment:
imo you can drop it. and just call go directly

Reply:
Removed